cmd/root.go

```diff
@@ -63,8 +63,10 @@ func initConfig() {
 		viper.SetConfigName(".sprinkler")
 	}
 
-	viper.AutomaticEnv()                                   // read in environment variables that match
-	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_")) // replaces "." with "_" for environment variables
+	viper.AutomaticEnv()                                      // read in environment variables that match
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))    // replaces "." with "_" for environment variables
+	viper.BindEnv("control.apiKeyEnabled", "API_KEY_ENABLED") // binds config key to environment variable
```

Comment:
why not use the auto generated env?

Reply:
The deploy scripts use API_KEY_ENABLED, XFCC_ENABLED.  Without BindEnv, it did not work.  

Maybe, it would bind APIKEYENABLED, XFCCENABLED without the underscore.  But that's hard to read.  I thought about changing the go code control vars to api_key_enabled, but that would different in convention from the other vars with camel-case convention.  I did not try actually test api_key_enabled renaming to know for sure though.